src/paging.md

```diff
@@ -216,9 +224,8 @@ this chart from top to bottom, each column is a case. So the first column says
 “if `a` is zero and `b` is zero, `or a b` will be zero.” The second column says
 “if `a` is one and `b` is zero, `or a b` will be one.” And so on.
 
-Now, we defined `p3_table` in the BSS section, which means that it will start as
-all zeroes. So when we `or` with `0b11`, it means that the first two bits will
-be set to one, keeping the rest as zeroes.
+So when we `or` with `0b11`, it means that the first two bits will be set to
```

Comment:
Why is this sentence removed?

Reply:
Because it's misleading. Or, at least, the way I read it misled me :)

It says:
> `p3_table` ... will start as all zeroes.
And:
> [W]e `or` with `0b11`... keeping the rest as zeroes.

But `p3_table` is not all zeroes. It's the address of the P3 table, which had better not be zero. The table itself, i.e. the chunk of memory that `p3_table` points to, is zeroes - but `p3_table` is not. It's definitely not true that we're "keeping the rest as zeroes" - the rest will be the address of the P3 table.